_observing-your-data/sso.md

```diff
@@ -0,0 +1,41 @@
+---
+layout: default
+title: Simple Schema for Observability
+nav_order: 120
+---
+
+# Simple Schema for Observability
+Introduced 2.6
+{: .label .label-purple }
+
+OpenSearch 2.6 introduced a standardization for conforming to a common and unified observability schema: Simple Schema for Observability (SSO). [Observability]({{site.url}}{{site.baseurl}}/observing-your-data/index/) is a collection of plugins and applications that let you visualize data-driven events by using Piped Processing Language (PPL) to explore and query data stored in OpenSearch.
+
+With the schema in place, Observability tools can ingest, automatically extract, and aggregate data and create custom dashboards, making it easier to understand the system at a higher level.
+
+SSO is inspired by both [OpenTelemetry](https://opentelemetry.io/docs/) and the Elastic Common Schema (ECS) and uses [ECS events](https://docs.aws.amazon.com/AmazonECS/latest/developerguide/ecs_cwe_events.html) for logs with the OpenTelemetry metadata.
```

Comment:
Unfortunately I'm still not following "uses ECS events for logs with the OpenTelemetry metadata." It also looks like we define ECS as Elastic Common Schema but then refer to ECS events, in which case ECS stands for Amazon Elastic Container Service (Amazon ECS). Let's rewrite this sentence so that ECS only has one meaning and so that we are only providing necessary information (does the reader really need to know what SSO was inspired by?).

Reply:
Re-wrote as per our conversation.